src/main.js

```diff
@@ -0,0 +1,92 @@
+"use strict";
```

Comment:
Let's try to not have compiled .js files in scr/ and just keep our ts in there. If you type tsc, it should compile everything and put the compiled .js files in dist/ folder

Reply:
Yeah, that's my bad. Usually, I manually delete the .js files.

tsc src/*.ts does not actually put the .js files in the ./dist folder. I don't know why. I've tried different edits on the TSConfig, but it doesn't work.

Reply:
Yeah I just ran tsc instead and it was working. It also compiles everything in experiments/ too but as long as none of those files are broken it's fine. But you have to modify the rootDirs field in tsconfig.json for that to work I believe. But the correct configuration is in the main branch I believe so no need to worry about it much.